src/ExpandObjects/epfilter.f90

```diff
@@ -23101,7 +23101,7 @@ SUBROUTINE CreateNewVRF
    ' HeatingEIRHiPLR')
   CALL AddToObjFld('Heating Combination Ratio Correction Factor Curve Name', base + vrfsNameOff,' HeatingCombRatio')
   CALL AddToObjFld('Heating Part-Load Fraction Correlation Curve Name', base + vrfsNameOff,' VRFCPLFFPLR')
-  CALL AddToObjStr('Minimum Heat Pump Part-Load Ratio','0.25')
+  CALL AddToObjFld('Minimum Heat Pump Part-Load Ratio', base + vrfsHPMinPLROff,'')
```

Comment:
Not sure why this was hard-wired if there is an input in the HVAC template object. This change uses the user input.